melissa/config.py

```diff
@@ -0,0 +1,31 @@
+# Melissa
+from profile import data
+import importlib
+import pkgutil
+import inspect
+
+dict_tts = dict()
+for (_, name, _) in pkgutil.iter_modules(["tts_engines"]):
+    module = importlib.import_module('.' + name + ".main", "tts_engines")
+    for cls, obj in inspect.getmembers(module):
+        if inspect.isclass(obj):
+            class_ = getattr(module, cls)
+            attributes = inspect.getmembers(class_)
+            li = [a for a in attributes if not(a[0].startswith('__') and
+                  a[0].endswith('__'))]
+            engine_name = li[0][1]
```

Comment:
Can't you just do `.name` instead of this scan? It's highly unreliable and unnecessary.